src/sentry/integrations/metric_alerts.py

```diff
@@ -173,9 +174,10 @@ def metric_alert_attachment_info(
     else:
         status = INCIDENT_STATUS[IncidentStatus.CLOSED]
 
-    url_query = None
+    url_query = {"detection_type": alert_rule.detection_type}
```

Comment:
🤔 - could we lookup the detection type on this page based on the alert id / notification uuid? (not super important just curious 😅 - i could also see using this for click thru rates and stuff like that)

Reply:
We could look it up with the alert rule id but why would we do that if we already have it here? And this is to be used for clickthrough rates